playground.js

```diff
@@ -0,0 +1,40 @@
+const form = document.getElementById("form");
+
+form.addEventListener("submit", function (e) {
+  e.preventDefault();
+  if (form === null) {
+    console.error("form element not found");
+    return;
+  }
+
+  const fd = new FormData(form);
+
+  const [replacement, remove, regex_g, regex_i, lowercase, trim] = Array.from(
+    fd.values()
+  );
+
+  const opts = {};
+
+  if (replacement.length === 1) {
```

Comment:
Muti-character replacements are supported.

```js
slug('a b c', {replacement: '///'}) // 'a///b///c'
```

So maybe this instead?

```suggestion
  if (replacement.length > 0) {
```